packages/ember-utils/lib/invoke.js

```diff
@@ -12,7 +15,7 @@ import applyStr from './apply-str';
   ```
 
   @method canInvoke
```

Comment:
`@static`

Reply:
I removed canInvoke - turns out it's not a package/module which makes sense because it is private API